walkthroughs/week-3-server/examples/random-quotes/src/main/webapp/index.html

```diff
@@ -10,7 +10,7 @@
     <div id="content">
       <h1>Random Quotes</h1>
       <p>Click here to see a random quote:</p>
-      <button onclick="getRandomQuote()">Fetch a quote!</button>
+      <button onclick="getRandomQuoteUsingArrowFunctions()">Fetch a quote!</button>
```

Comment:
Did you mean to include this change?

Reply:
Yes, it was a step of the Intro to Fetch tutorial, to see how two functions written very differently can work the same.

Reply:
![image](https://user-images.githubusercontent.com/45107360/86178804-dc9c7000-badd-11ea-8d46-8919e87fd0fa.png)
I just saw this comment, will apply to this as well :)

Reply:
Yeah it's not the end of the world if this ends up in the PR, but I think in general the examples files don't need to be in the PR!